curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-oop-by-building-a-platformer-game/65b2bb4c279af3cd585ba777.md

```diff
@@ -21,10 +21,10 @@ You should have a checkpoint detection rule that checks for the following: `play
 assert.match(code, /player\.position\.x\s*-\s*player\.width\s*<=\s*checkpoint\.position\.x\s*-\s*checkpoint\.width\s*\+\s*player\.width\s*\*\s*0\.9/i);
 ```
 
-You should have a checkpoint detection rule that checks for the following: `index === 0 || checkpoints[index - 1].claimed === true`.
+You should have a checkpoint detection rule that checks for the following: `index === 0 || checkpoints[index - 1].claimed === true` or `index === 0 || checkpoints[index - 1].claimed`.
```

Comment:
Let's revert this change for now. 
It is outside of the scope of this particular PR.
It is best to contain all related changes into one PR and then open up separate ones for other changes.
Plus there is probably are some other changes that can be made to this wording and tests but that should be handled in a separate issue and PR.